pkg/identities/service.go

```diff
@@ -163,9 +168,55 @@ func (s *Service) CreateIdentity(ctx context.Context, bodyID *kClient.CreateIden
 
 	s.authz.SetCreateIdentityEntitlements(ctx, identity.Id)
 
+	err = s.SendUserCreationEmail(ctx, identity)
+
 	return data, err
 }
 
+func (s *Service) SendUserCreationEmail(ctx context.Context, identity *kClient.Identity) error {
+	ctx, span := s.tracer.Start(ctx, "identities.Service.SendUserCreationEmail")
+	defer span.End()
+
+	template, err := mail.LoadTemplate(mail.UserCreationInvite)
+	if err != nil {
+		return err
+	}
+
+	code, link, err := s.generateRecoveryInfo(ctx, identity.Id)
+	if err != nil {
+		return err
+	}
+
+	emailAddress := identity.Traits.(map[string]interface{})["email"].(string)
```

Comment:
```suggestion
	if e, ok := identity.Traits.(map[string]interface{})["email"]; ok {
emailAddress = e.(string)
}
```
reason is that the schema might have email under something named different I guess

Reply:
Mmmm this could be a problem since we need a way to obtain that address. I'll apply your suggestion and let's talk at today's daily about this. Thanks